Framework/TestHelpers/inc/MantidTestHelpers/NexusFileReader.h

```diff
@@ -10,17 +10,19 @@
 
 #include "MantidNexusGeometry/NexusGeometryDefinitions.h"
 
+#include <Eigen/Dense>
 #include <H5Cpp.h>
 #include <boost/filesystem.hpp>
 #include <string>
 #include <vector>
 
 /*
- TODO: DOCUMENTATION
-
- *@author Takudzwa Makoni, RAL (UKRI), ISIS
- *@date 06/08/2019
-*/
+ * NexusFileReader: Test utility for unit testing in
+ * NexusGeometrySave::saveInstrument.
+ *
+ * @author Takudzwa Makoni, RAL (UKRI), ISIS
+ * @date 06/08/2019
```

Comment:
We typically don't include author and date information in files, that is what Git is for.